framework/origin_trials_client.py

```diff
@@ -151,7 +151,7 @@ def create_origin_trial(ot_stage: Stage) -> str | None:
         f'Failed to get response from origin trials API. {response.text}')
     raise e
 
-  return response.json()['id']
+  return str(response.json()['id'])
```

Comment:
Modify the tests for it to reflect the data type

Reply:
Done!